vizro-core/examples/features/yaml_version/app.py

```diff
@@ -2,14 +2,20 @@
 
 from pathlib import Path
 
+import numpy as np
 import vizro.plotly.express as px
 import yaml
 from vizro import Vizro
 from vizro.managers import data_manager
 from vizro.models import Dashboard
 
+tips = px.data.tips()
+tips["smoker"] = np.where(tips["smoker"] == "Yes", 1, 0)
 data_manager["iris"] = px.data.iris()
 data_manager["gapminder_2007"] = px.data.gapminder().query("year == 2007")
+data_manager["gapminder"] = px.data.gapminder()
+data_manager["tips"] = tips
+
```

Comment:
Here is a suggestion for resorting data preparation code lines, similar like in the comment above.
```suggestion
tips = px.data.tips()
tips["smoker"] = np.where(tips["smoker"] == "Yes", 1, 0)
data_manager["tips"] = tips
gapminder = px.data.gapminder()
data_manager["gapminder"] = gapminder
data_manager["gapminder_2007"] = gapminder.query("year == 2007")
data_manager["iris"] = px.data.iris()
```

Reply:
Yes! - Thank you! 🚀 🧹